arm-resources/resources/2016-09-01/swagger/resources.json

```diff
@@ -1853,6 +1853,7 @@
           }
         },
         "debugSetting": {
+          "x-ms-client-flatten": true,
```

Comment:
@jhendrixMSFT - Can you please update the PR by removing the x-ms-client-flatten extension? Once that is done, the PR can be merged.